core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -503,6 +530,16 @@ private DefaultFunctionResolver dayOfYear(BuiltinFunctionName dayOfYear) {
     );
   }
 
+  private DefaultFunctionResolver extract() {
+    return define(BuiltinFunctionName.EXTRACT.getName(),
+        impl(nullMissingHandling(DateTimeFunction::exprExtract), LONG, STRING, DATE),
+        impl(nullMissingHandling(DateTimeFunction::exprExtract), LONG, STRING, DATETIME),
+        impl(nullMissingHandling(DateTimeFunction::exprExtract), LONG, STRING, TIME),
```

Comment:
TIME requires `FunctionProperties` and convert to datetime (see `extractDateTime`)

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/221/commits/d15c26ba044d8db36dad2dcf197757404aeb8ece. Modified again in https://github.com/Bit-Quill/opensearch-project-sql/pull/221/commits/4dccd2358d4f372469933ef1d5deea9125b4e890